doc/conf.py

```diff
@@ -3,12 +3,11 @@
 This configuration only affects single-package Sphinx documentation builds.
 """
 
-from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.ts.ess
+from documenteer.conf.pipelinespkg import *  # noqa
+import lsst.ts.salobj  # noqa
```

Comment:
Please change salobj to ess (I made that mistake in most of my packages and had to scramble to correct it. I'm sorry if my bad example triggered this.)